Makefile

```diff
@@ -120,15 +80,10 @@ run-etcd:
 	--advertise-client-urls "http://$(LOCAL_IP_ENV):2379,http://127.0.0.1:2379" \
 	--listen-client-urls "http://0.0.0.0:2379"
 
-semaphore:
-	# Ensure Semaphore has loaded the required modules
-	modprobe -a ip6_tables xt_set
-
-	# Run the STs
-	make st
-
+semaphore: test-containerized
 	set -e; \
 	if [ -z $$PULL_REQUEST_NUMBER ]; then \
+		$(MAKE) libnetwork-plugin.created; \
```

Comment:
More a general Makefile approach question.  For Docker builds, do we really gain much from having .created files?  Provided nothing else has changed re-running the Docker build is generally pretty quick, so we could just let Docker handle the build decision...